diff --git a/src/main/java/com/scaleiq/processor/stream/processor1.java b/src/main/java/com/scaleiq/processor/stream/processor1.java
index 3eb0c2f..e34d219 100644
--- a/src/main/java/com/scaleiq/processor/stream/processor1.java
+++ b/src/main/java/com/scaleiq/processor/stream/processor1.java
@@ -36,8 +36,11 @@ import java.util.Properties;
 public class processor1 {
 
     public static RestHighLevelClient createOpenSearchClient() {
-        String connString = "http://localhost:9200";
-//
+        String connString = "https://admin:admin@localhost:9200";
+
+        //Point to keystore with appropriate certificates for security.
+        System.setProperty("javax.net.ssl.trustStore", "/home/soundarya/cubescale/assessment/ssl-configs/mytruststore.jks");
+        System.setProperty("javax.net.ssl.trustStorePassword", "password123");
 
         // we build a URI from the connection string
         RestHighLevelClient restHighLevelClient;
